docs/getting-started.md

```diff
@@ -155,124 +109,15 @@ these environment variables in your own way.
 > the OS keystore as permanent storage, and offers shell features to securely
 > expose and setup local AWS environments.
 
```

Comment:
It seems like we should have actual steps here rather than deferring to the cluster-api docs only.

Reply:
I'd rather use this as a pre-requisite and have the documentation in one place, rather than scattered around in multiple ones, wdyt?

Reply:
It just seems odd to call this a getting started guide and not provide users with instructions on how to get started.

Reply:
@detiber which capi docs?

Reply:
@ncdc there is a link to the cluster-api book on line 3

Reply:
Options:

1. Have CAPI as a prerequisite and provide a link to the CAPI quickstart
1. Copy the CAPI quickstart text here (just enough to get CAPI + CABPK + CAPA working) and maintain in both places

I wanted to say that I prefer option 1. However, looking at the rendered page (https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/d816bdb1618a067d80a407213b88a6616d780118/docs/getting-started.md), if you read the content from top to bottom, this is not a "getting started" guide. It's unclear how to get a cluster up and running in AWS.

Maybe instead of this being a "getting started" doc, it's more of a doc about CAPA prereqs and troubleshooting?

Reply:
Let's chat about it at the meeting today?